embeddedtest/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/suborbital/sat/sat"
+)
+
+// Set the following environment variables before running
+// 	SAT_CONTROL_PLANE={Control Plane URL}
+// 	SAT_ENV_TOKEN={Environment token}
+//
+// Replace Fully Qualified Function Name in sat.ConfigFromRunnableArg to load a function
+// See also https://docs.suborbital.dev/compute/concepts/fully-qualified-function-names
+func main() {
```

Comment:
Maybe we should rename this embedded example or something like that. It's not really self-contained nor is it really a test. 

If we decide to rename it example maybe take in the FQFN as an argument as opposed to the hard coded value from the POC